frontend/src/layout/navigation/MainNavigation.tsx

```diff
@@ -232,7 +237,13 @@ export function MainNavigation(): JSX.Element {
                     />
                     <div className="divider" />
                     {canViewPlugins(user?.organization) && (
-                        <MenuItem title="Plugins" icon={<ApiFilled />} identifier="plugins" to="/project/plugins" />
+                        <MenuItem
+                            title="Plugins"
+                            icon={<ApiFilled />}
+                            identifier="plugins"
+                            to="/project/plugins"
+                            highlight={!user?.flags['has_checked_out_plugins']}
```

Comment:
Tutorials are kind of tricky. Do we want to highlight plugins who sign up 2 weeks from now? For them this won't be new code.

Reply:
Not relevant as this has been removed.